package.json

```diff
@@ -27,17 +27,21 @@
     "posttest": "yarn run lint"
   },
   "engines": {
-    "node": ">=10.10.0"
+    "node": ">=10.14.2"
   },
   "peerDependencies": {
     "@babel/core": "^7.0.0",
     "typescript": "^3.0.0"
   },
+  "resolutions": {
+    "typescript-to-proptypes/typescript": "^3.9.6"
```

Comment:
Unfortunately `typescript-to-proptypes` uses a specific version of `typescript` (3.6 I belive). And that didn't support some features in the `tsconfig.json` retrieved from `@milesj/build-tools`